Userland/Libraries/LibWeb/OutOfProcessWebView.cpp

```diff
@@ -245,6 +245,11 @@ void OutOfProcessWebView::notify_server_did_change_title(Badge<WebContentClient>
         on_title_change(title);
 }
 
+void OutOfProcessWebView::notify_server_did_request_scroll(Badge<WebContentClient>, int wheel_delta)
+{
+    vertical_scrollbar().set_value(vertical_scrollbar().value() + wheel_delta * 20);
```

Comment:
This is a bit yucky... just copied from `ScrollableWidget::handle_wheel_event`, but maybe this should be factored out (and that way the multiplication constant can be more sensibly defined)?

Also, is it sufficient to just give a wheel delta? I guess we don't have horizontal/vertical wheel scrolling to differentiate right now?